src/modules/launcher/Plugins/Microsoft.Plugin.Folder/InternalDirectoryProcessor.cs

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Plugin.Folder.Sources;
+using Microsoft.Plugin.Folder.Sources.Result;
+using Wox.Plugin;
+
+namespace Microsoft.Plugin.Folder
+{
+    public class InternalDirectoryProcessor : IFolderProcessor
+    {
+        private readonly IFolderHelper _folderHelper;
+        private readonly IQueryInternalDirectory _internalDirectory;
+
+        public InternalDirectoryProcessor(IFolderHelper folderHelper, IQueryInternalDirectory internalDirectory)
+        {
+            _folderHelper = folderHelper;
+            _internalDirectory = internalDirectory;
+        }
+
+        public IEnumerable<IItemResult> Results(string actionKeyword, string search)
+        {
+            if (!_folderHelper.IsDriveOrSharedFolder(search))
+            {
+                return Enumerable.Empty<IItemResult>();
+            }
+
+            return _internalDirectory.Query(search);
+        }
```

Comment:
@P-Storm can you please clarify why the `actionKeyword` was added to the Results function in both the processors, ie. the `InternaldirectoryProcessor` and  `UserFolderProcessor`. It is not being used in the functions though. Thanks! 

PS - I know the PR is merged in but we're working on the dev docs and I came across this. Thanks again.

Reply:
I did remove it, but didn't remove it at the function. I did think implementing it with > was needed for recursive search, but at the end I have dropped it. So it can be safely removed.

Reply:
Great! thanks for the clarification.